liblouis/compileTranslationTable.c

```diff
@@ -2707,12 +2889,101 @@ compileRule(FileInfo *file, TranslationTableHeader **table,
 		}
 		return putCharDotsMapping(
 				file, ruleChars.chars[0], ruleDots.chars[0], displayTable);
+	case CTO_None: {
+		// check if token is a macro name
+		if (inScopeMacros) {
+			const MacroList *macros = *inScopeMacros;
+			while (macros) {
```

Comment:
I find this a bit hard to read. A for loop that loops through the list might be more readable

Reply:
How would that look?